python/invoke_release/tasks.py

```diff
@@ -1045,6 +1049,67 @@ def _post_rollback(current_version, rollback_to_version):
         plugin.post_rollback(ROOT_DIRECTORY, current_version, rollback_to_version)
 
 
+def _get_version_to_bump(changelog_message):
+    major_commit_present = None
+    minor_commit_present = None
+    patch_commit_present = None
+
+    for line in changelog_message:
+        if line.startswith('- ' + MAJOR_TAG):
```

Comment:
could we have a MAJOR_TAG_PREFIX = '- [MAJOR]' for clarity?

Reply:
I agree, that is probably best.